CHANGELOG.md

```
@@ -1,6 +1,42 @@
+
+- Add export for generateProfileUrl ([159f025](https://github.com/kinde-oss/js-utils/commit/159f025))
+
+### 🏡 Chore
+
+- Release v0.16.0 ([9b1ed67](https://github.com/kinde-oss/js-utils/commit/9b1ed67))
+
+### ❤️ Contributors
+
+- Daniel Rivers ([@DanielRivers](https://github.com/DanielRivers))
+
+## 0.16.0...feat/generateProfileUrl
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)